ispdb/iiyama-catv.ne.jp.xml

```diff
@@ -4,18 +4,27 @@
     <domain>iiyama-catv.ne.jp</domain>
     <displayName>ケーブルテレビiネット飯山</displayName>
     <displayShortName>iネット飯山</displayShortName>
+    <incomingServer type="imap">
+      <hostname>imap.iiyama-catv.ne.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILLOCALPART%</username>
```

Comment:
issue: the documentation states (in the yellow table, right below SMTP settings, I'm translating through DeepL):

> Username or account name: Email address of the guidance notice.

Which I understand to mean we should use the whole email address. Same comment applies to POP3 and SMTP.